easybuild/easyconfigs/g/gettext/gettext-0.19.8.eb

```diff
@@ -23,4 +23,16 @@ dependencies = [('ncurses', '6.0')]
 
 configopts = '--without-emacs --with-included-libxml --without-xz --without-bzip2'
 
-moduleclass = 'vis'
+sanity_check_paths = {
+    'files': ['bin/gettext', 'lib/libasprintf.a', 'lib/libasprintf.%s' % SHLIB_EXT,
+              'lib/libgettextpo.a', 'lib/libgettextpo.%s' % SHLIB_EXT],
+    'dirs': ['include'],
+}
+
+sanity_check_paths = {
+    'files': ['bin/gettext', 'lib/libasprintf.a', 'lib/libasprintf.%s' % SHLIB_EXT,
+              'lib/libgettextpo.a', 'lib/libgettextpo.%s' % SHLIB_EXT],
+    'dirs': ['include'],
+}
```

Comment:
@boegel - cut and paste error? Assume they're not supposed to be duplicated...

Reply:
Indeed, thanks for the thorough review!

I blame it on Friday afternoon...